clientv3/retry.go

```diff
@@ -27,12 +27,30 @@ type retryRpcFunc func(context.Context, rpcFunc) error
 
 func (c *Client) newRetryWrapper() retryRpcFunc {
 	return func(rpcCtx context.Context, f rpcFunc) error {
+		nrAuthRetries := 0
+
 		for {
 			err := f(rpcCtx)
 			if err == nil {
 				return nil
 			}
 
+			if grpc.Code(err) == codes.Unauthenticated {
```

Comment:
OK, I'll change the condition. And retrying during the ctx is valid seems to be reasonable.